docs/user/cli/export.rst

```diff
@@ -124,3 +124,53 @@ have all index types.
 
 The ``--gzip`` argument can be used to compress the output through **gzip** encoding. It can be specified
 as a number between 1-9. Higher numbers indicate more compression, lower numbers indicate faster compression.
+
+``playback``
+------------
+
+The playback command can simulate a streaming ingestion by replaying features that have already been ingested.
+Features are returned based on a date attribute in the feature. For example, if replaying three features that
+have dates that are each one second apart, each feature will be emitted after a delay of one second. The rate
+of export can be modified to speed up or slow down the original time differences.
+
+In order to simulate a data stream, the output of this command can be piped into another process, for example
+to send messages to a Kafka topic or write files to NiFi for ingestion.
+
+======================== =========================================================
+Argument                 Description
+======================== =========================================================
+``-c, --catalog *``      The catalog table containing schema metadata
+``-f, --feature-name *`` The name of the schema
+``--interval *``         Date interval to replay, in the format
+                         ``yyyy-MM-dd'T'HH:mm:ss.SSSZ/yyyy-MM-dd'T'HH:mm:ss.SSSZ``
+``--dtg``                Date attribute to base playback on. If not specified,
+                         will use the default schema date field
+``--rate``               Rate to speed-up features being returned, as a float
```

Comment:
[minor] The detailed text below indicates that this can be used to *slow* the rate as well, which means the brief description could be considered inaccurate.

Reply:
fixed